### DISCUSSION TITLE
preferred team penalty gif's

### DISCUSSION THREAD

**Discussion by posh-man** [riffnshred/nhl-led-scoreboard#273] [Dev and Ideas]
Hello All,

I have been loving this project. Great work to everyone who has made this possible. I am wondering if we can add the preferred team goal idea to the penalty function. This way we could have other gif's playing depending on powerplay or short handed situations. I see the folder structure is there but I was looking at the code in src/renderer/main.py and it looks like the penalty function for picking other gifs is not available now. If I wanted to write something in for my own board and test it out this is file is the right one? Should be in around line 339 or so? If this is on the road map then I will wait if not I will tinker with it a bit.

Thanks,
Scott

**Comment by riffnshred** [comment]
Hey Scott, sorry for the delay,

There is currently a bug with the animations in the master. I got it fixed in the beta branch and it will be part of the next update. see issue #256 for details.
